src/txn/txn_rollback_to_stable.c

```diff
@@ -759,7 +836,7 @@ __rollback_page_needs_abort(
      * greater than the given timestamp:
```

Comment:
This comment needs an update according to the recovered snapshot and newest_txn on the page.

Reply:
Updated comments.